api/client_test.go

```diff
@@ -254,7 +254,7 @@ func TestDefaulRetryPolicy(t *testing.T) {
 				t.Fatalf("expected to retry request: '%t', but actual result was: '%t'", test.expect, retry)
 			}
 			if err != test.expectErr {
-				t.Fatalf("expected error from retry policy: '%s', but actual result was: '%s'", err, test.expectErr)
+				t.Fatalf("expected error from retry policy: %q, but actual result was: %q", err, test.expectErr)
```

Comment:
There's a slight difference between these two in that any quoted strings within those values will be escaped if using `%q` but not with `'%s'`, but if the spirit of the output here is to quote the error message itself and not so much about escaping quoted values within `err` then I think this is fine to do.

https://go.dev/play/p/xZQeqklVxRq

Reply:
oh that's great to know! I'd imagine most/all of these cases were from not knowing/thinking about using the `%q` verb, but if it looks like there was some other intention it shouldn't be changed